src/Impostor.Server/Net/Inner/Objects/Components/InnerPlayerPhysics.cs

```diff
@@ -1,22 +1,35 @@
 ﻿using System;
 using System.Threading.Tasks;
 using Impostor.Api;
+using Impostor.Api.Events.Managers;
 using Impostor.Api.Net;
 using Impostor.Api.Net.Messages;
 using Impostor.Server.Net.State;
+using Impostor.Server.Events.Player;
 using Microsoft.Extensions.Logging;
 
 namespace Impostor.Server.Net.Inner.Objects.Components
 {
     internal partial class InnerPlayerPhysics : InnerNetObject
     {
         private readonly ILogger<InnerPlayerPhysics> _logger;
+        private readonly IEventManager _eventManager;
         private readonly InnerPlayerControl _playerControl;
+        private readonly Game _game;
+        private bool _isWatchingCamera;
+        public bool IsWatchingCamera { get {return _isWatchingCamera; }
```

Comment:
Might be better suited for PlayerControl instead of PlayerPhysics, but I'm not sure.

Reply:
We need to consider different options.
I chose `PlayerPhysics`  because I think it is similar to Vent logic there.
`PlayerPhysics` can be extended with these : `InVent`, `IsDoingTask`, etc.